lib/request-proxy.js

```diff
@@ -59,7 +59,7 @@ proto.proxyReq = function proxyReq(opts) {
     var timeout = opts.timeout ?
         opts.timeout : ringpop.proxyReqTimeout;
 
-    body(req, onBody);
+    body(req, res, { cache: true }, onBody);
 
     function onBody(err, rawBody) {
```

Comment:
yea, this is a hot path that'll need careful attn. ill fix it in a different PR.